devicesetup/src/main/java/io/particle/android/sdk/devicesetup/SetupCompleteIntentBuilder.java

```diff
@@ -0,0 +1,9 @@
+package io.particle.android.sdk.devicesetup;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.Nullable;
+
+public interface SetupCompleteIntentBuilder {
+    Intent buildIntent(Context ctx, @Nullable SetupResult result);
```

Comment:
I'd like to see this be Parcelable. I know that's a little awkward for implementation, but doing so is a way to guarantee that we can avoid leaks. I'd bet that most implementations of this interface will be anonymous inner classes from activities, which means we will leak.

thoughts?